src/Stan/Inspection.hs

```diff
@@ -57,13 +78,20 @@ inspections =
         { inspectionId = Id "STAN-0001-HEAD"
         , inspectionName = "Partial: base/head"
         , inspectionDescription = "Usage of partial function 'head' for lists"
-        , inspectionSolution = unlines
-            [ "* Replace list with 'NonEmpty' from 'Data.List.NonEmpty'"
-            , "* Use explicit pattern-matching over lists"
+        , inspectionSolution =
+            [ "Replace list with 'NonEmpty' from 'Data.List.NonEmpty'"
+            , "Use explicit pattern-matching over lists"
             ]
         , inspectionCategory =
             -- TODO: See issue #25: https://github.com/kowainik/stan/issues/25
             one $ Category "Partial"
         , inspectionSeverity = Severe
         }
     ]
+
+-- | Get the 'Inspection' by the given inspection 'Id'.
+getInspectionById :: Id Inspection -> Inspection
+getInspectionById insId = case find ((==) insId . inspectionId) inspections of
+    Just ins -> ins
+    -- TODO: how to handle unknown ids better?
```

Comment:
It sounds like a difficult problem... Let's create an issue and discuss it? Ideally, I'd like to have a mapping from `Id Inspection` to `Inspection` where we use known `inspectionId`. But I'm not sure we can convince the compiler to do this efficiently...

Reply:
See https://github.com/kowainik/stan/issues/31